__tests__/__integration__/cmd/src/cli/auto-format/object/Object.definition.ts

```diff
@@ -9,7 +9,7 @@
 *
 */
 
-import { ICommandDefinition } from "../../../../../../../packages/index";
+import { ICommandDefinition } from "../../../../../../../lib/index";
```

Comment:
curious as to why we specify `/index` in the path only for some files and not all.

Reply:
Not sure 🙂 For this PR I wanted to keep the amount of changes minimal. Perhaps in the future we should evaluate all of our imports?